ui/test/dashboards/reducers/ui.test.ts

```diff
@@ -58,15 +58,35 @@ const d2 = {...dashboard, id: 2, cells: [], name: 'd2', templates: []}
 const dashboards = [d1, d2]
 
 describe('DataExplorer.Reducers.UI', () => {
+  beforeEach(() => {
+    state = undefined
+  })
+
   it('can load the dashboards', () => {
-    const actual = reducer(state, loadDashboards(dashboards, d1.id))
+    const actual = reducer(state, loadDashboards(dashboards))
     const expected = {
       dashboards,
     }
 
     expect(actual.dashboards).toEqual(expected.dashboards)
   })
 
+  it('can load dashbaords into existing dashboards state', () => {
+    const newTemplate = {id: '3', ...template}
+    const updatedD1 = {...d1, templates: [...d1.templates, newTemplate]}
+
+    state = {
```

Comment:
[I notice this file also uses this style to setup a `loadedState`:](https://github.com/influxdata/chronograf/pull/3823/files#diff-3fcefe1f15da2f483ae2cae5e348233bL78)